charts/mod-lists/values.yaml

```diff
@@ -113,6 +113,8 @@ extraEnv: |
     value: {{ .Values.listExportBatchSize | quote }}
   - name: spring.task.execution.pool.max-size
     value: {{ .Values.springTaskPoolSize | quote }}
+  - name: LIST_APP_S3_BUCKET
+    value: {{ .Values.awsConnectParameters | quote }}
```

Comment:
This approach won't work, we've tried to build the same chart locally and result is not the same as expected one:
![image](https://github.com/folio-org/folio-helm-v2/assets/138369232/643574b2-53f6-42de-bf91-52ac62f31167)

Reply:
Expected result, should like below one:
![image](https://github.com/folio-org/folio-helm-v2/assets/138369232/fce781ad-20bf-43a9-b922-8aef345533a8)